cmake/CMakejucer.cmake

```diff
@@ -81,4 +94,20 @@ function(jucer_project_end)
     ${JUCER_PROJECT_INCLUDE_DIRS}
   )
 
+  if(APPLE)
+    target_compile_options(${JUCER_PROJECT_NAME} PRIVATE -std=c++11)
+    target_compile_definitions(${JUCER_PROJECT_NAME} PRIVATE
+      $<$<CONFIG:Debug>:_DEBUG>
+      $<$<NOT:$<CONFIG:Debug>>:NDEBUG>
```

Comment:
Defined twice indeed.


-------------------


`CompileC build/HelloWorld.build/Release/HelloWorld.build/Objects-normal/x86_64/MainComponent.o Source/MainComponent.cpp normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
    cd /Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld
    export LANG=en_US.US-ASCII
    /usr/local/opt/ccache/libexec/clang -x c++ -arch x86_64 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -Wno-trigraphs -fpascal-strings -O3 -Wno-missing-field-initializers -Wno-missing-prototypes -Wno-return-type -Wno-non-virtual-dtor -Wno-overloaded-virtual -Wno-exit-time-destructors -Wno-missing-braces -Wparentheses -Wswitch -Wno-unused-function -Wno-unused-label -Wno-unused-parameter -Wno-unused-variable -Wunused-value -Wno-empty-body -Wno-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wno-constant-conversion -Wno-int-conversion -Wno-bool-conversion -Wno-enum-conversion -Wno-shorten-64-to-32 -Wno-newline-eof -Wno-c++11-extensions -DCMAKE_INTDIR=\"Release\" -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk -fasm-blocks -fstrict-aliasing -Wdeprecated-declarations -Winvalid-offsetof -mmacosx-version-min=10.9 -Wno-sign-conversion -I/Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/Release/include -I/Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/JuceLibraryCode -I/Users/neogaldr/Unsaved/Developments/JUCE.cmake/../JUCE/examples/HelloWorld/../../modules -I/Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/HelloWorld.build/Release/HelloWorld.build/DerivedSources/x86_64 -I/Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/HelloWorld.build/Release/HelloWorld.build/DerivedSources -Wmost -Wno-four-char-constants -Wno-unknown-pragmas -F/Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/Release -DNDEBUG -std=c++11 -MMD -MT dependencies -MF /Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/HelloWorld.build/Release/HelloWorld.build/Objects-normal/x86_64/MainComponent.d --serialize-diagnostics /Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/HelloWorld.build/Release/HelloWorld.build/Objects-normal/x86_64/MainComponent.dia -c /Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/Source/MainComponent.cpp -o /Users/neogaldr/Unsaved/Developments/JUCE.cmake/examples/HelloWorld/build/HelloWorld.build/Release/HelloWorld.build/Objects-normal/x86_64/MainComponent.o`